ADR/0048-movable-build-tests.md

```diff
@@ -0,0 +1,68 @@
+# 48. Attestable Build-Time Tests in Integration Service
+
+Date: 2025-04-06
+
+## Status
+
+Proposed
+
+## Context
+
+Some of the security scans run during the build pipeline take a long time to
+run. This forces users to wait longer to be notified of the results of their
+builds.  In order to receive more immediate feedback, users have requested that
+they be able to run security scans during their integration tests.
+
+## Decision
+
+We will add support for adding attestations to images from their integration
+pipelines. Test pipelines can be run and re-run independently of build
+pipelines but all pipelines will have attestations generated for them to enable
+Conforma evaluations.
+
+### Attestations
+
+Each task will be responsible for generating an attestation in Cosign. `vuln`
```

Comment:
Are you trying to say https://github.com/in-toto/attestation/blob/main/spec/predicates/vuln.md or https://github.com/in-toto/attestation/blob/main/spec/predicates/vulns_02.md?

Reply:
We might also want other types of attestations, i.e. https://github.com/in-toto/attestation/blob/main/spec/predicates/test-result.md

Reply:
0.2, I'll update the document with a link.